From fbd5c7a78d2fe2c56795d05193c5d22b3a2863cd Mon Sep 17 00:00:00 2001
From: Your Name <zenobit@disroot.org>
Date: [20240110]
Subject: [PATCH] Change man page installation directory

* meson.build (configure_file): Update install directory for man pages.

Fixes: [Reference to the issue or reason for the change, if applicable]
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index abcdef1..1234567 100644
--- a/meson.build
+++ b/meson.build
@@ -123,7 +123,7 @@ configure_file(
     output: 'wayfire.1',
     install: true,
-    install_dir: 'man',  # Change this line to install in /usr/share/man
+    install_dir: 'share/man',
     configuration: conf_data
 )

-- 
2.20.1
